cmake/target/src/target_config.c

```diff
@@ -36,6 +36,7 @@
 #include "cfe_mission_cfg.h"
 #include "cfe_platform_cfg.h"
 #include "cfe_es.h"
+#include "cfe_es_table.h"
```

Comment:
Shouldn't need to include this file here.

Reply:
oh actually this is needed for the "CFE_PLATFORM_ES_TBL_FILE" definition...

Reply:
Isn't that it `cfe_platform_cfg.h`?

Reply:
correct, corrected :)